integrations/services/communications/message.py

```diff
@@ -43,7 +43,10 @@ def email(self, email: str, send_tests=False):
         content = Content("text/html", self._email_body())
         mail = Mail(from_email, to_email, subject, content)
 
-        sg.client.mail.send.post(request_body=mail.get())
+        try:
+            sg.client.mail.send.post(request_body=mail.get())
+        except Exception as e:
+            print("e:", e)
```

Comment:
I think that we should just let this error fall through.
```suggestion
        sg.client.mail.send.post(request_body=mail.get())
```